### PR TITLE
Support for `db_out`

### PR DESCRIPTION
**Pull request by JulienR1**
Accepts incoming payloads generated by `db_out` (https://github.com/streamingfast/substreams-sink-database-changes).

~~TODO: Remove `./src/tmp.ts` and replace with the updated versions of `substreams-sink-database-changes` and `substrams-sink-webhook`.~~

### CODE REVIEW

**Comment by DenisCarriere** [comment]
👍  looks good, published npm package for DatabaseChanges

File: src/clickhouse/handleSinkRequest.ts [hunk 1]
```diff
@@ -86,59 +91,73 @@ function batchSizeLimitReached() {
...
+    const values = getValuesInEntityChange(change);
+    handleChange(change.entity, values, change.operation, { ...metadata, id: change.id });
+  }
+}
+
+function handleDatabaseChanges(tableChanges: TableChange[], metadata: Metadata) {
+  logger.info(`handleSinkRequest | tableChanges=${tableChanges.length}`);
+  for (const change of tableChanges) {
+    const values = getValuesInTableChange(change);
+    handleChange(change.table, values, change.operation, { ...metadata, id: "" });
```
Comment (JulienR1) [inline src/clickhouse/handleSinkRequest.ts:106]: Should the `id` be empty or the primary key in `db_out`?

Comment (DenisCarriere) [reply src/clickhouse/handleSinkRequest.ts:106]: Oh that's a good question... the primary key in `db_out` is `pk` or `compositePk`

**Review by DenisCarriere — APPROVED** [review]
👍  looks good

**Pull request merged**